pyproject.toml

```diff
@@ -16,3 +16,9 @@ exclude = '''
     | __pycache__
 )/
 '''
+
+[tool.isort]
```

Comment:
Shed should configure isort already, isn't it?

Reply:
Yes, shed configures it, this is needed for my code editor, isort is triggered on every file save and produces result that is different from shed. Should I use `.isort.cfg` config locally and not commit it to github?